src/test/java/software/amazon/awssdk/crt/test/SigningTest.java

```diff
@@ -440,6 +440,23 @@ private HttpRequest createChunkedTestRequest() throws Exception {
         return new HttpRequest("PUT", uri.getPath(), requestHeaders, null);
     }
 
+    private HttpRequest createChunkedTrailerTestRequest() throws Exception {
+
+        URI uri = new URI("https://s3.amazonaws.com/examplebucket/chunkObject.txt");
+
+        HttpHeader[] requestHeaders =
+                new HttpHeader[]{
+                        new HttpHeader("Host", uri.getHost()),
+                        new HttpHeader("x-amz-storage-class", "REDUCED_REDUNDANCY"),
+                        new HttpHeader("Content-Encoding", "aws-chunked"),
+                        new HttpHeader("x-amz-decoded-content-length", "66560"),
+                        new HttpHeader("Content-Length", "66824"),
+                        new HttpHeader("x-amz-trailer", "first,second,third")
```

Comment:
I don't understand aws-chunked, just throw questions instead of diving deep for now.
Per my understand of trailer, there will be headers after body right? But that seems not have anything like that. Just double check to make sure that's not a miss.

Reply:
These are regular headers, the trailing headers are added elsewhere "first:1st;second:2nd;third:third". x-amz-trailer header lists the trailing headers to expect.